development-styleguide.md

```diff
@@ -11,12 +11,55 @@ Furthermore, it shows commons mistakes or scenarios and how to handle them
 - Such practice should generally be avoided as it can lead to maintenance problems and can produce hard to find bugs
 - If you don't own it, don't modify it
 
-
 ```typescript
-let bob = { name : "Bob", surname: "Smith", age : 23 }
+let bob = { name: "Bob", surname: "Smith", age: 23 };
 bob.age = 24; // avoid
 // create a new object using spread operator to update only specific fields
-const olderBob = { ...bob, age: 24 }
+const olderBob = { ...bob, age: 24 };
+```
+
+#### Avoid hiding exceptions
+
+Usually in Javascript and Typescript is hard to figure out whether a function throws an exception or not. Try to be explicit about the exceptions you throw.
```

Comment:
I'd add: "try not to use exceptions at all, you can use eithers instead in 99% of cases. Exceptions must be limited to the scope of the main() method in case you need to early exit the process with an error message"